### ISSUE TITLE
Python environment to run "How To Guides"

### ISSUE THREAD

**Issue by RMeli** [pytorch-ignite/examples#76]
In the `how-to-guides` sometimes dependencies are assumed, sometimes they are installed with
```
!pip install
```
I suggest to uniform this by providing a `conda` and/or `requirements.txt` file/s so that a full environment can be created.

**Comment by vfdev-5** [comment]
@RMeli thanks for the suggestion! If we opt to provide a `requirements.txt` where we can store it such that it would be accessible, especially if a notebook is open with colab ?
We can think of adding a step to download it (for example with wget) from somewhere and then install it. However, wget solution may not always work on Win OS...
Maybe, `!pip install dep1 dep2` could be more simple and cross-platform. What do you think ?

**Comment by RMeli** [comment]
OIC, is the assumption that most of those notebooks run on Google Colab? That explains why some packages are not installed, I believe they are readily available.

> Maybe, !pip install dep1 dep2 could be more simple and cross-platform. What do you think ?

Yes, if everything is pip-installable then it would probably be the easiest/safest solution.

**Comment by vfdev-5** [comment]
> is the assumption that most of those notebooks run on Google Colab? 

Yeah, sort of implicit assumption as very easy to run notebooks on Colab
